### DISCUSSION TITLE
Make CSP build available via npm registry

### DISCUSSION THREAD

**Discussion by doompadee** [alpinejs/alpine#2395] [4. General]
The documentation seems to imply that the CSP compliant build is already available via registry, while it still is not: https://alpinejs.dev/advanced/csp

Any chance to have it uploaded soon? Thanks.

**Comment by MichaelMackus** [comment]
Definitely, would be very useful! Anything we can do to help @calebporzio  ?

**Comment by AndyMillne** [comment]
Yes please, this would be a nice update.

**Comment by karltynan** [comment]
Yes, this is something I definitely require as I do not understand how to "_build it from source_" as mentioned here in the docs: https://alpinejs.dev/advanced/csp - Either some guidance on how to achieve this or the NPM version would be amazing.

**Reply by ekwoka** [reply]
You just clone the repo, install deps, and run the build script

**Comment by Alercard** [comment]
I tried the alpine csp build solution but It didn't work! I still having the issue! 
Could somebody help me with any direction to solve this CSP issue?
![imagen](https://github.com/alpinejs/alpine/assets/8834939/ddc1214f-dc58-4145-bc22-19a5b0d687c6)
![imagen](https://github.com/alpinejs/alpine/assets/8834939/9b829761-be6f-4783-aa9e-aeef22a0df84)

**Reply by ekwoka** [reply]
This was built from the CSP build?

Can you verify that?

**Reply by Alercard** [reply]
I did every step detailed in https://alpinejs.dev/advanced/csp and I used module.esm.js in js files.

The cloned https://github.com/alpinejs/alpine repository and I built the main branch.

Do you think it is ok?
